coverage-targets.js

```diff
@@ -6,10 +6,10 @@
 // subset of files to check against these targets.
 module.exports = {
   global: {
-    lines: 70.94,
-    branches: 59.11,
-    statements: 70.38,
-    functions: 63.65,
+    lines: 70.85,
```

Comment:
Could you please show me where this number is coming from? Should we revert this value back to before as 
```
    lines: 71.15,
    branches: 59.17,
    statements: 70.6,
    functions: 63.82,
```

Reply:
Seems to be the required values in circleCI job https://app.circleci.com/pipelines/github/MetaMask/metamask-extension/47180/workflows/250b5f5c-6d72-4716-9126-a23b47d986a8/jobs/1343665

Reply:
Clear. hope this works !